mappings/net/minecraft/entity/passive/PolarBearEntity.mapping

```diff
@@ -19,7 +19,7 @@ CLASS net/minecraft/class_1456 net/minecraft/entity/passive/PolarBearEntity
 	METHOD method_6602 playWarningSound ()V
 	METHOD method_6603 setWarning (Z)V
 		ARG 1 warning
-	CLASS class_1457 FollowPlayersGoal
+	CLASS class_1457 PlayerTargetGoal
```

Comment:
This one is like targeting to protect baby (when there is baby close around) while the other one is general "anger at" target goal